partner_data_vies_populator/tests/test_partner_data_vies_populator.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (C) 2015 Forest and Biomass Romania
+# Copyright (C) 2020 NextERP Romania
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import Form, TransactionCase
+
+
+class TestPartnerCreateByVAT(TransactionCase):
+    def setUp(self):
+        super().setUp()
```

Comment:
Could you switch to class method ?

Reply:
@rousseldenis And it's done. Thanks for your review :)